spec/test_app/app/routing/posts_routing_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+describe 'routes for posts', type: :routing do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.